### PR TITLE
More informative KeyError messages in converter

### PR DESCRIPTION
**Pull request by Ayxan13**
Added a try...catch to `get_member(self, name)` method of `ConverterObject`.

### CODE REVIEW

**Comment by heinezen** [comment]
Nice ty

**Review by heinezen — APPROVED** [review]

File: openage/convert/entity_object/conversion/converter_object.py [hunk 1]
```diff
@@ -68,7 +68,10 @@ def get_member(self, name):
         """
         Returns a member of the object.
         """
-        return self.members[name]
+        try:
+            return self.members[name]
+        except KeyError as err:
+            raise KeyError(f"{self} has no attribute: {key}") from err
```
Comment (heinezen) [inline openage/convert/entity_object/conversion/converter_object.py:74]: change `{key}` to `{name}` pls

**Review by heinezen — CHANGES_REQUESTED** [review]

**Comment by heinezen** [comment]
And you need to put your name + mail in `copying.md`

File: openage/convert/entity_object/conversion/converter_object.py (hunk 1 above)
Comment (Ayxan13) [reply openage/convert/entity_object/conversion/converter_object.py]: Fixed it. Thank you for the catch!

**Review by heinezen — APPROVED** [review]

**Comment by heinezen** [comment]
Merged!

**Pull request merged**